include/chainblocks.hpp

```diff
@@ -59,7 +59,7 @@ class InvalidVarTypeError : public CBException {
   explicit InvalidVarTypeError(std::string_view msg) : CBException(msg) {}
 };
 
-CBlock *createBlock(std::string_view name);
+CHAINBLOCKS_API CBlock *createBlock(std::string_view name);
```

Comment:
We actually want to export ONLY chainblocksInterface and consume from `CBCore` for many reasons - we could talk about in a issue or so. But in a nutshell it's for versioning and coherence, so those are not necessary